### ISSUE TITLE
5G

### ISSUE THREAD

**Issue by rubenhaz** [shinydevelopment/SimulatorStatusMagic#114]
Will there be an update with 5G options? 4G is already a thing of the past in a lot of places.

**Comment by chrisvasselli** [comment]
Good point! I think most people are just using the defaults which don't show the connected network (matching the style of Apple's screenshots), so it hasn't come up before. It's probably a matter of just adding another entry to `SDStatusBarManagerNetworkType` and verifying that it updates the interface correctly.

I'll take a look when I can, or if you'd like to give it a try and create a PR, that might be a little faster. 😊
